src/Microsoft.AspNetCore.Mvc.TagHelpers/PartialTagHelper.cs

```diff
@@ -78,10 +82,31 @@ public object Model
         /// <summary>
         /// When optional, executing the tag helper will no-op if the view cannot be located. 
         /// Otherwise will throw stating the view could not be found.
+        /// Cannot be used together with <see cref="Fallback"/>.
         /// </summary>
         [HtmlAttributeName(OptionalAttributeName)]
         public bool Optional { get; set; }
 
+        /// <summary>
+        /// View to lookup if main view cannot be located.
+        /// Will throw if main view and fallback cannot be located.
+        /// Cannot be used together with <see cref="Optional"/>.
```

Comment:
Is there a specific reason to make this choice? Would it make sense to just not throw in general if `optional` is set to keep the behavior predictable?

Reply:
Thank you for the review! This is misunderstanding from my part. Yes, it would indeed make more sense to not throw if optional is set.